src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -104,4 +121,79 @@ const ChartsHolder = (props: {
   );
 };
 
+function caseGrowthStatusText(projection: Projection) {
+  const rt = projection.rt!;
+  if (rt === null) {
+    return 'No case load data is available.';
+  }
+  const level = determineZone(CASE_GROWTH_RATE, rt);
```

Comment:
@igorkofman This is similar to what you'd want to do for the homepage recoloring.  We get the indicator out of the projection (`projection.rt` in this case) and then use `determineZone()` to see if it's `Level.Low`, `Level.Medium`, or `Level.High`.  You'd need to do that for each of the key indicators (see below 2 functions) and then compare to find the highest one, and then feed that to https://github.com/covid-projections/covid-projections/blob/2e0f17bda1fa7f412c584cbe76e843ec0ac45bea/src/enums/zones.ts#L23 to get the color.

Reply:
Oh, and you'll need to either merge this PR first or branch off of it, since I added `currentTestPositiveRate` and `currentIcuUtilization`, which you'll need.